### PR TITLE
Fix clippy warnings. Add proper names to symboltables.

### PR DESCRIPTION
**Pull request by windelbouwman**

### CODE REVIEW

**Comment by windelbouwman** [comment]
@coolreader18 or @palaviv  could one of you take a look at this change, or should I request someone else review?

**Comment by coolreader18** [comment]
Apologies, I'll review.

File: vm/src/scope.rs [hunk 1]
```diff
@@ -142,6 +143,12 @@ impl NameProtocol for Scope {
         self.load_global(vm, name)
     }
 
+    #[cfg_attr(feature = "flame-it", flame("Scope"))]
+    /// Load a local name. Only check the local dictionary for the given name.
+    fn load_local(&self, vm: &VirtualMachine, name: &str) -> Option<PyObjectRef> {
+        self.get_locals().get_item_option(name, vm).unwrap()
+    }
```
Comment (coolreader18) [inline vm/src/scope.rs]: Looks nice, this should be really fast in theory.

**Comment by windelbouwman** [comment]
No worries, I just wanted to have somebody to take a look at it. Thanks!

**Pull request merged**